SIDFactoryII/source/runtime/emulation/sid/sidproxy.cpp

```diff
@@ -102,15 +109,48 @@ namespace Emulation
 
 			const double passband = 20000.0;
 
-			m_pSID->setSamplingParameters
-			(
+			double filter_8580_curve = GetSingleConfigurationValue<Utility::Config::ConfigValueFloat>(Global::instance().GetConfig(), "Sound.Emulation.8580.FilterCurve", 0.5);
+			if (filter_8580_curve < 0.0)
```

Comment:
Probably a good idea to move this clamping code to `configfile.h` or `configfile.cpp`? 
Same code is also duplicated for `Window.Scaling`

Reply:
I don't agree with that, but it would make sense to make a utility for this which may also do the output for the log.